### ISSUE TITLE
tests fail for v3/integrations/nrgrpc

### ISSUE THREAD

**Issue by pymnh** [newrelic/go-agent#344]
Hi. For v3.14.0, tests for v3/integrations/nrgrpc are failing with following error:
```
% go test
--- FAIL: TestUnaryServerInterceptorError (0.00s)
    expect_implementation.go:27: error events value difference key=error.class gRPC Status: DataLoss gRPC Error: DataLoss
--- FAIL: TestStreamServerInterceptorError (0.00s)
    expect_implementation.go:27: error events value difference key=error.class gRPC Status: DataLoss gRPC Error: DataLoss
FAIL
exit status 1
FAIL	github.com/newrelic/go-agent/v3/integrations/nrgrpc	0.048s
```

**Comment by nr-swilloughby** [comment]
Thanks for the heads up. We'll look into that and also why the tests passed for us but failed later when installed.

**Comment by nr-swilloughby** [comment]
That was a typo in the testcase. It's been corrected in our development branch and will be released ASAP.

**Comment by nr-swilloughby** [comment]
This has been fixed in the 3.14.1 go-agent release (nrgrpc integration 1.3.1)
